lab_1_classify_by_unigrams/main.py

```diff
@@ -80,6 +169,21 @@ def preprocess_profile(profile: dict) -> dict[str, str | dict] | None:
     :return: a dict with a lower-cased loaded profile
     with relative frequencies without unnecessary ngrams
     """
+    if not (isinstance(profile, dict)
+        and 'name' in profile
+        and 'freq' in profile
+        and 'n_words' in profile
+    ):
+        return None
+
+    unigram_profile = {'name': profile['name'], 'freq': {}}
+    for token in profile['freq']:
+        if token.lower() in unigram_profile['freq']:
+            unigram_profile['freq'][token.lower()] += profile['freq'][token] / profile['n_words'][0]
+        elif len(token) == 1:
```

Comment:
Can we only use one condition here? Without `else`? Hint: try to firstly check for the length

Reply:
honestly, i don't know how to rewrite it:(